keystore-manager.sh

```
@@ -282,6 +354,771 @@ encode_base64() {
+    # Read MATCH_PASSWORD from .match_password file if it exists
+    local MATCH_PWD=""
+    if [[ -f "secrets/.match_password" ]]; then
+        MATCH_PWD=$(head -n1 secrets/.match_password 2>/dev/null | tr -d '\n\r')
+        print_success "Loaded MATCH_PASSWORD from .match_password file"
+    else
+        print_warning "secrets/.match_password not found - MATCH_PASSWORD will be empty"
+    fi
+
+    # Extract values from shared_keys.env
+    # Format: export KEY="value"
+    local APPSTORE_KEY_ID=$(grep '^export APPSTORE_KEY_ID=' "$SHARED_KEYS_FILE" 2>/dev/null | cut -d'"' -f2)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)